test/method.getEmailDetails.test.js

```diff
@@ -46,7 +46,8 @@ tap.test('getEmailDetails - with yaml', (assert) => {
       data: {
         firstName: 'bob',
         lastName: 'smith',
-        serviceName: 'test city'
+        serviceName: 'test city',
+        trackingPixel: ''
```

Comment:
I added a default for the tracking pixel of a '' so that if it is used in a template it is not trying to use an undefined variable. So now it is an empty string.